clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClient.groovy

```diff
@@ -49,6 +51,9 @@ class DockerRegistryClient {
   @Value('${dockerRegistry.client.timeout:60000}')
   int clientTimeout
 
+  @Value('${dockerRegistry.client.paginateSize:100}')
+  int paginateSize
```

Comment:
@tomaslin, I actually can't seem to get this value to be read properly. Regardless of if I have a `dockerRegistry.client.paginateSize` field in my config, it's always set to 0. I had to manually override it for testing. Any ideas?